app/consapp/rtkrcv/conf/rtk_skylark_l1l5.conf

```diff
@@ -70,7 +70,7 @@ pos2-armode        =wide-lane  # (0:off,1:continuous,2:instantaneous,3:fix-and-h
 # SWIFT: only enable for static positioning mode
 pos2-wlarmode      =off        # (0:off,1:on)
 pos2-gloarmode     =off        # (0:off,1:on)
-pos2-bdsarmode     =on         # (0:off,1:on)
+pos2-bdsarmode     =off        # (0:off,1:on)
```

Comment:
nit, I guess here it depends on the customers' need. With the change, it defaults to no BDS AR.

Reply:
Correct.  The whole point of this PR is to disable BDS AR when widelane is enabled.

Reply:
Alternatively we can enable `pos2-bdsarmode` by default but remove BDS from `pos1-navsys`.  Then in the App Note I can add a comment saying to disable `pos2-bdsarmode` if BDS is added to `pos1-navsys`.

Reply:
If we don't plan on using BDS, i think we can remove BDS from pos1-navsys. With the current configuration as off, it means BDS will still be used in the float solution, as the second case in the following.
1 RTK with no BDS
2 RTK with BDS in float mode
3 RTK with BDS in fixed mode